### PR TITLE
Feature/cicd setup

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
This PR is to set up CI to build ROS2 code following ROS2 merge to develop

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+version: 2
+#  Copyright (C) 2018-2021 LEIDOS.
```
Comment (msmcconnell) [inline .circleci/config.yml:2]: 2018-2022

File: docker/install.sh [hunk 2]
```diff
@@ -16,11 +16,12 @@
...
 source /home/carma/catkin/setup.bash
+source /opt/ros/foxy/setup.bash
 # Install dependencies
 sudo apt-get update
 rosdep update
 rosdep install --from-paths src --ignore-src -r -y
 # Build 
-colcon build --packages-up-to novatel_oem7_driver carma_novatel_driver_wrapper
\ No newline at end of file
+colcon build --packages-up-to novatel_oem7_driver carma_novatel_driver_wrapper
```
Comment (msmcconnell) [inline docker/install.sh:26]: no need for the --packages-up-to
do
```
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
```
instead

File: docker/install.sh [hunk 3]
```diff
@@ -16,11 +16,12 @@
...
+#colcon build --packages-up-to novatel_oem7_driver
```
Comment (msmcconnell) [inline docker/install.sh:27]: commented out thing can be deleted

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply .circleci/config.yml]: Updated

File: docker/install.sh (hunk 2 above)
Comment (SaikrishnaBairamoni) [reply docker/install.sh]: Updated

File: docker/install.sh (hunk 3 above)
Comment (SaikrishnaBairamoni) [reply docker/install.sh]: Updated

**Review by msmcconnell — APPROVED** [review]

**Pull request merged**